Src/Private/Get-AbrFgtSystem.ps1

```diff
@@ -250,43 +250,204 @@ function Get-AbrFgtSystem {
-                            #"Speed"              = $interface.speed
-                        }
-                    }
+                    foreach ($group in $groupedInterfaces) {
+                        $interfaceType = $group.Name
 
-                    $TableParams = @{
-                        Name         = "Interface"
-                        List         = $false
-                        ColumnWidths = 12, 20, 7, 11, 6, 8, 28, 8
-                    }
+                        # Create a heading for each interface type
+                        Section -Style Heading4 "$interfaceType Interfaces" {
+                            $OutObj = @()
 
-                    if ($Report.ShowTableCaptions) {
-                        $TableParams['Caption'] = "- $($TableParams.Name)"
-                    }
+                            foreach ($interface in $group.Group) {
+
+                                # Standardise interface properties
+                                $interface.name = $interface.name + $($interface.alias ? "`n($($interface.alias))" : "")
+                                $interface.role = $interface.role -eq 'undefined' ? "" : ($interface.role).ToUpper()
+                                $interface.member = $interface.member.count -gt 0 ? $interface.member.'interface-name' -join ', ' : ""
+                                $interface.mtu = $interface.'mtu-override' -eq 'disable' ? '' : $interface.mtu
+                                $interface.mode = $interface.mode -eq 'static' ? '' : $interface.mode
+                                $interface.ip = $interface.ip -eq '0.0.0.0 0.0.0.0' ? '' : $interface.ip
+                                $interface.'secondaryip' = if ($interface.'secondary-ip' -eq 'enable' -and $null -ne $interface.'secondaryip') {
+                                    ($interface.'secondaryip' | ForEach-Object {
+                                        $_.ip
+                                    }) -join ', '
+                                } else {
+                                    ""
+                                }
+                                $interface.mode = $interface.mode -eq 'static' ? '' : $interface.mode
+                                $interface.vdom = $interface.vdom -eq 'root' ? '' : $interface.vdom
+                                $interface.vlanid = ($interface.vlanid -gt 0 ) ? $interface.vlanid : ""
+                                $interface.speed = $interface.speed -eq 'auto' ? '' : $interface.speed
+                                $interface.'remote-ip' = $interface.'remote-ip' -eq '0.0.0.0 0.0.0.0' ? '' : $interface.'remote-ip'
+
+
+                                switch ($interfaceType) {
+                                    "Aggregate" {
+                                        $OutObj += [pscustomobject]@{
+                                            "Name"                = $interface.name
+                                            "VDOM"                = $interface.vdom
```

Comment:
really need to display vdom ? because by default you select the vdom...

Reply:
For multi-vdom fortigates, this will be useful. If there's only a single one, this column will not be displayed anyway.